src/pykeen/nn/modules.py

```diff
@@ -477,11 +477,7 @@ def _prepare_state_for_functional(self) -> MutableMapping[str, Any]:
         return dict()
 
 
-class NormBasedInteraction(
-    FunctionalInteraction,
-    Generic[HeadRepresentation, RelationRepresentation, TailRepresentation],
-    ABC,
-):
+class NormBasedInteraction(Interaction, Generic[HeadRepresentation, RelationRepresentation, TailRepresentation], ABC):
```

Comment:
this change has a lot of impact on other models besides TransE

Reply:
Yes. I was thinking about making the changes for all NormBased models at once , but first wanted to see what breaks ;-)